src/languages/tap.js

```diff
@@ -22,7 +22,7 @@ export default function(hljs) {
       },
       // YAML block
       {
-        begin: '(\s+)?---$', end: '\\.\\.\\.$',
+        begin: /---$/, end: '\\.\\.\\.$',
```

Comment:
This was broken before `\s` vs `\\s` and when fixed seemed to break things so without any context going to go with the simpler rule for now.